src/main/java/io/boomerang/controller/WorkflowController.java

```diff
@@ -79,9 +80,11 @@ public void deleteToken(@PathVariable String id, @RequestParam String label) {
   public FlowWorkflowRevision getWorkflowLatestVersion(@PathVariable String workFlowId) {
     final WorkflowEntity entity = workFlowRepository.getWorkflow(workFlowId);
 
-    if (entity.getScope() == WorkflowScope.user
-        && !entity.getOwnerUserId().equals(userIdentityService.getCurrentUser().getId())) {
-      throw new HttpClientErrorException(HttpStatus.FORBIDDEN);
+    if (!UserType.admin.equals(userIdentityService.getCurrentUser().getType())) {
+      if (entity.getScope() == WorkflowScope.user
+          && !entity.getOwnerUserId().equals(userIdentityService.getCurrentUser().getId())) {
+        throw new HttpClientErrorException(HttpStatus.FORBIDDEN);
+      }
```

Comment:
@tlawrie can we combine these if blocks into one?

  `if (!UserType.admin.equals(userIdentityService.getCurrentUser().getType())
        && entity.getScope() == WorkflowScope.user
        && !entity.getOwnerUserId().equals(userIdentityService.getCurrentUser().getId())) {
      throw new HttpClientErrorException(HttpStatus.FORBIDDEN);
    }`

Reply:
Yes, yes we can.